<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DenoisingSettingsWidget</class>
 <widget class="QWidget" name="DenoisingSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <widget class="Line" name="upperHorizontalLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="denoisingHorizontalLayout">
     <item>
      <widget class="QLabel" name="denoisingLabel">
       <property name="text">
        <string>Denoising method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="denoisingComboBox">
       <item>
        <property name="text">
         <string>No filter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bilateral filter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Short duration transient suppression filter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Noise gate filter</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="denoisingStackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="bilateralPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0" colspan="2">
        <layout class="QVBoxLayout" name="bilateralPreviewVerticalLayout"/>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="bilateralParametersHorizontalLayout">
         <item>
          <layout class="QGridLayout" name="bilateralLeftGridLayout">
           <item row="1" column="1">
            <widget class="QLineEdit" name="sigmaSpaceLineEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="sigmaColorLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="sigmaSpaceLabel">
             <property name="text">
              <string>Sigma space</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="sigmaColorLabel">
             <property name="text">
              <string>Sigma color</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="bilateralVerticalLine">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="bilateralRightGridLayout">
           <item row="0" column="1">
            <widget class="QLineEdit" name="dLineEdit"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="restoreBilateralButton">
             <property name="text">
              <string>Restore defaults</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="dLabel">
             <property name="text">
              <string>d</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="bilateralMiddleHorizontalLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SDTSPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="SDTSParametersHorizontalLayout" stretch="1,1,1">
         <item>
          <layout class="QGridLayout" name="SDTSLeftGridLayout">
           <item row="0" column="2">
            <widget class="QSlider" name="placeholderSDTSNoiseDecreaseSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="SDTSNoiseDecreaseLabel">
             <property name="text">
              <string>Noise decrease</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="mLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="mLabel">
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="SDTSVerticalLine">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="SDTSverticalLayout">
           <item>
            <widget class="QFrame" name="SDTSFillerFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="restoreSDTSButton">
             <property name="text">
              <string>Restore defaults</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="Line" name="SDTSMiddleHorizontalLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QVBoxLayout" name="SDTSPreviewVerticalLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="noiseGatePage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="noiseGateParametersHorizontalLayout">
         <item>
          <layout class="QGridLayout" name="noiseGateLeftGridLayout">
           <item row="2" column="1">
            <widget class="QLineEdit" name="nStdThreshLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="nGradTimeLabel">
             <property name="text">
              <string>Gradient pixels number (time)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="nGradFreqLabel">
             <property name="text">
              <string>Gradient pixels number (frequency)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="noiseGateNoiseDecreaseLabel">
             <property name="text">
              <string>Noise decrease</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="nGradTimeLineEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="nStdThreshLabel">
             <property name="text">
              <string>Number std cutoff</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSlider" name="placeholderNoiseGateNoiseDecreaseSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nGradFreqLineEdit"/>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="noiseGateVerticalLine">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="noiseGateRightGridLayout">
           <item row="4" column="1">
            <widget class="QLabel" name="noiseStartLabel">
             <property name="text">
              <string>Noise start</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLineEdit" name="noiseEndLineEdit"/>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="noiseStartLineEdit"/>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QCheckBox" name="noiseAudioCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Use main spectrogram as noise source</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLabel" name="noiseLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QPushButton" name="restoreNoiseGateButton">
             <property name="text">
              <string>Restore defaults</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QPushButton" name="loadNoiseAudioButton">
             <property name="text">
              <string>Load audio with noise</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="noiseEndLabel">
             <property name="text">
              <string>Noise end</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="noiseGateMiddleHorizontalLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="noiseGatePreviewVerticalLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="noFilterPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
